Sources/PointFree/RunnerTasks/ValidateEnterpriseEmails.swift

```diff
@@ -0,0 +1,92 @@
+import Either
+import Mailgun
+import Models
+import PointFreePrelude
+import Prelude
+
+public func validateEnterpriseEmails() -> EitherIO<Error, Prelude.Unit> {
```

Comment:
the whole process of validating enterprise emails is broken into a bunch of steps